nengo/config.py

```diff
@@ -139,16 +133,13 @@ def __init__(self, configures, clsparams):
     def __contains__(self, key):
         return self in self._clsparams.get_param(key)
 
-    def __getattribute__(self, key):
-        try:
-            return super(InstanceParams, self).__getattribute__(key)
-        except AttributeError:
-            if key in self._clsparams.default_params:
-                raise
-            param = self._clsparams.get_param(key)
-            if self in param:
-                return param.__get__(self, self.__class__)
-            return getattr(self._clsparams, key)
+    def __getattr__(self, key):
+        if key in self._clsparams.default_params:
+            raise AttributeError()
```

Comment:
I'm not totally sure why this is an error, so I'm not totally sure what to put. Before, it just got passed through from the `super(InstanceParams, self).__getattribute__(key)` call, so it would say something like "AttributeError: 'InstanceParams' object has no attribute 'radius'". Is this what we want?